challenge3/webapp/msaito/app/controllers/statics_controller.rb

```diff
@@ -0,0 +1,31 @@
+class StaticsController < ApplicationController
+  before_action :search
+  helper_method :sort_column, :sort_direction 
+
+  def index
+    @houses = House.search(params).page(params[:page]).per(10).order(sort_column + " " + sort_direction)
+  end
+
+  def show
+    @house = House.find(params[:id])
+    @datas = @house.datasets.pluck(:year,:month, :temperature, :daylight, :energy_production)
```

Comment:
`pluck` 使っているのは good

Reply:
```suggestion
    @datas = @house.datasets.pluck(:year, :month, :temperature, :daylight, :energy_production)
```
空白の使い方がそろってないところがある。